bin/wasm-node/rust/src/lib.rs

```diff
@@ -468,6 +470,14 @@ async fn handle_rpc(rpc: &str, client: &mut Client) -> (String, Option<String>)
                 .to_json_response(request_id);
             (response, None)
         }
+        methods::MethodCall::author_submitExtrinsic { transaction } => {
+            let response = match announce_transaction(client, transaction.0).await {
+                Ok(transaction_hash) => methods::Response::author_submitExtrinsic(transaction_hash)
+                    .to_json_response(request_id),
+                Err(e) => todo!("{:?}", e), //TODO:
```

Comment:
There's a `build_error_response` or something method used below that can be used for inspiration.